sbi/simulators/simutils.py

```diff
@@ -77,3 +79,27 @@ def simulate_in_batches(
         x = simulator(theta)
 
     return x
+
+
+@contextlib.contextmanager
+def tqdm_joblib(tqdm_object):
+    """Context manager to patch joblib to report into tqdm progress bar given as
+    argument
+
+    This wrapped context manager obtains the number of finished tasks from the tqdm
+    print function and uses it to update the pbar.
```

Comment:
Maybe refer to issue #419 in the docstring? Otherwise the existence of this function seems a bit cryptic.